_data/rico-news.yml

```diff
@@ -0,0 +1,14 @@
+- title: 'Blog post "Introducing Rico"'
```

Comment:
is it possible to do this automatically by filtering via a rico tag?\

//cc @hendrikebbers @giftkugel